### PR TITLE
Improve the instruction for releasing marketplaces

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Updated the document describing the repository release 

**Related issue(s)**
See also #35

### CODE REVIEW

**Review by pPrecel — APPROVED** [review]
Great work! Thanks for this update!

File: docs/releases.md [hunk 1]
```diff
@@ -1,62 +1,61 @@
...
 
     ```bash
     git push upstream 1.1.0
     ```
 
-8. Monitor the release job.
-9. Check release notes on [GitHub.com](https://github.com/kyma-incubator/marketplaces/releases).
+7. Monitor the status of the release job on the [Prow dashboard](https://status.build.kyma-project.io/?job=rel-marketplaces).
+
+8. Check if the [release notes](https://github.com/kyma-incubator/marketplaces/releases) on were published on GitHub.
```
Comment (mmitoraj) [inline docs/releases.md]:
```suggestion
8. Check if the [release notes](https://github.com/kyma-incubator/marketplaces/releases) were published on GitHub.
```

**Review by mmitoraj — APPROVED** [review]
One comment :)

File: docs/releases.md [hunk 2]
```diff
@@ -1,62 +1,61 @@
...
     ```
 
-6. Create a tag with the proper release version:
+4. Create a release branch:
 
     ```bash
-    git tag 1.1.0
-    ```
+   git checkout -b release-1.1
+   git push upstream release-1.1
```
Comment (hudymi) [inline docs/releases.md]: The problem is when you are doing it from scratch (from point 1) then you should push to origin. Same in point 6

**Review by hudymi — APPROVED** [review]
One comment, but I followed instruction and release 1.8.0 is ready ;)

**Pull request merged**